src/test/java/uk/gov/hmcts/reform/et/syaapi/service/AcasServiceTest.java

```diff
@@ -33,7 +34,8 @@
 class AcasServiceTest {
 
     private static final String ACAS_DEV_API_URL = "https://api-dev-acas-01.azure-api.net/ECCLDev";
-    private static final String ACAS_API_KEY = "ecd18c987120415b95f60b844e6730ef";
+    @Value("${acas.api.key}")
+    private String ACAS_API_KEY;//= "27703f6efe064209ac619eb51dbabefd";
```

Comment:
Please remove the ACAS API key comment. We shouldn't put any key references in code.